client/controllers/incidentReports/suggestedIncidentModal.coffee

```diff
@@ -4,10 +4,9 @@ incidentReportSchema = require '/imports/schemas/incidentReport.coffee'
 { stageModals } = require '/imports/ui/modals'
 
 Template.suggestedIncidentModal.onRendered ->
-  instance = @
```

Comment:
In `onRendered`? I think there was only one?
I did found [this](https://github.com/ecohealthalliance/eidr-connect/blob/2343ba7a6bd8721b54f551c87e06c11f77d6d2f7/client/controllers/incidentReports/suggestedIncidentModal.coffee#L29) which should be `incident._id`